packages/mobile/src/send/Send.tsx

```diff
@@ -189,6 +186,9 @@ function Send({ route }: Props) {
       <SendHeader isOutgoingPaymentRequest={isOutgoingPaymentRequest} />
       <DisconnectBanner />
       <SendSearchInput input={searchQuery} onChangeText={throttledSearch} />
+      {inviteRewardsEnabled && numberVerified && hasGivenContactPermission && (
+        <InviteRewardsBanner />
+      )}
```

Comment:
Curious, why moving moving this here?

Reply:
oh! sorry, i should have left a comment. i only want to dispatch the impression event once (this is done inside the InviteRewardsBanner component) but the renderListHeader is called multiple times on mount so i was getting too many events. the UI still looks the same if the banner lives here, so i opted to do this rather than to move the invites analytics event into this Send component since it does so much already. hope you agree :D